File: 3110-score-of-a-string/3110-score-of-a-string.java
class Solution {
    public int scoreOfString(String s) {
        int left = 0, score = 0, size = s.length();

        for(int right = 1; right < size; right++) {
            int lVal = s.charAt(left), rVal = s.charAt(right);
            score += lVal > rVal ? lVal - rVal : rVal - lVal;
            left++;
        }

        return score;
    }
} 

File: 3110-score-of-a-string/3110-score-of-a-string.c
int scoreOfString(char* s) {
    int left = 0, score = 0;

    for(int right = 1 ; s[right] != '\0' ; right++) {
        score += s[left] > s[right] ? s[left] - s[right] : s[right] - s[left];
        left++;
    }

    return score;
}